<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCustomPlot" name="widget" native="true">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>552</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="count_down_label">
          <property name="text">
           <string>Count Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="count_down">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="jump_down">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Jump Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jump_down_value_label">
          <property name="text">
           <string>Jump Down Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="jump_down_value">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="threshold_down_label">
          <property name="text">
           <string>Threshold Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="threshold_down">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-128</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>2</y>
         <width>528</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="count_up_label">
          <property name="text">
           <string>Count Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="count_up">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="jump_up">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Jump Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jump_up_value_label">
          <property name="text">
           <string>Jump Up Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="jump_up_value"/>
        </item>
        <item>
         <widget class="QLabel" name="threshold_up_label">
          <property name="text">
           <string>Threshold Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="threshold_up">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>127</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>592</x>
         <y>2</y>
         <width>278</width>
         <height>59</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="execution_rate_label">
            <property name="text">
             <string>Execution Rate (ms)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="execution_rate">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="execution_rate_label_2">
              <property name="text">
               <string>Periods</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fault_periods">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="total_failt_time_label">
            <property name="text">
             <string>Total Fail Time (ms)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="total_fail_time">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>6000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
